server.js

```diff
@@ -21,35 +21,37 @@ app.use(express.static('public'));
 // body parser config to accept our datatypes
 app.use(bodyParser.urlencoded({ extended: true }));
 
+var db = require('./models');
```

Comment:
Since you are requiring the entire models folder, you have access to ALL the models inside of it. In this case that means "book.js" so you can say `db.Book` to get access to that model.